src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -1263,7 +1267,7 @@ public function renderFieldAction($productId, $step, $fieldName)
             $this->get('prestashop.adapter.data_provider.tax'),
             $this->get('router')
         );
-        $form = $this->createFormBuilder($modelMapper->getFormData());
+        $form = $this->createFormBuilder($modelMapper->getFormData($product));
```

Comment:
We need to check this with QA 😮 I would be surprised / afraid it does not modify the behavior

Reply:
I don't understand why `getFormData` don't return any error as it lacks a parameter

Reply:
Magic !

Reply:
No problem. After validation, I ask a QA with check on BO & Product ;)